Topics/Tech_Stacks/Unity_UI.md

```diff
@@ -0,0 +1,107 @@
+# A Beginner's Guide for Unity UI Design
+
+### Table of contents
+#### [Introduction](#introduction)
+#### [Start with Canvas](#start-with-canvas)
+#### [Add UI element](#add-ui-element)
+#### [Hierarchy: Order of UI element](#hierarchy-order-of-ui-element)
+#### [Customize your UI element!](#customize-your-ui-element)
+#### [How fancy the Unity UI can be](#how-fancy-the-unity-ui-can-be)
+#### [Additional resources](#additional-resources)
+#### [References](#references)
+
+## Introduction
+
+Unity is one of the most powerful tool for game development while the User Interface (UI) 
+provides the most direct way to interact and guide users. Unity has its own Uinity UI system that allows game developers to create dynamic and interactive User Interface design while can also integrate into the game fast and quick.
+
+Here is a guidance for beginners to start designing their UI in Unity. For more details, please go and check the official [Unity Documentations](https://docs.unity3d.com/2023.3/Documentation/Manual/UIToolkits.html) for the version that you use.
+
+## Start with Canvas
+
+The Canvas is the place where you create and design your UI. It is independent from the actually game scene design and all other Game Objects, and you will find it always on top your other 3D objects or 2D sprite. However, what is cool about Unity UI is taht you can connect the UI with the actually game through game logic scripts.
```

Comment:
I'm not really sure what it means by "you will find it always on top your other 3D objects or 2D sprite"
Do you mean the canvas is in a different layer than the other game objects in the scene view?
Maybe you can include a screenshot to show what the canvas looks like in scene view and the relation of the canvas and other game objects in the scene view

Reply:
Great suggestion! So i added a picture here to show the relationship between the Canvas and the Game objects. Do you it is much clear?
Also i am not sure how to rephrase "you will find it always on top your other 3D objects or 2D sprite" to make it easier to understand. Do you have any ideas on this?

Reply:
I think the picture and explanations under it were good.
For rephrasing, I'm just not entirely sure what you are trying to say in this sentence. Do you mean the canvas folder is one of the folder in the top layer in the Hierarchy section? Or do you mean the items in canvas will appear on top of everything else in the game scene?

Reply:
So the canvas folder will not always be the top in Hierarchy. What I mean is the latter, items in the canvas will appear on the top of everything else in the game scene.